core/silkworm/chain/config.cpp

```diff
@@ -16,8 +16,19 @@
 
 #include "config.hpp"
 
+#include <functional>
+
+#include <silkworm/common/as_range.hpp>
+
 namespace silkworm {
 
+static const std::vector<std::pair<std::string, const ChainConfig*>> kKnownChainConfigs{
```

Comment:
Why not `std::map`?

Reply:
A little bit easier to deal with case insensitive comparator on the name